src/TorchSharp/Hub.cs

```diff
@@ -24,7 +24,8 @@ public static void set_create_progress_bar_func(Func<bool, IProgressBar> func)
                 createProgressBarFunc = func;
             }
 
-            internal static IProgressBar _create_progress_bar(bool hidden)
+            // Used internally by TorchVision
+            public static IProgressBar _create_progress_bar(bool hidden)
```

Comment:
We can't have a public method start with a '_' -- if this is public only because it's needed in TV, maybe it can be a) duplicated or b) moved to some 'internal' or 'implementation' namespace somewhere.

Reply:
Renamed to `CreateProgressBar()`. It cannot be duplicated unless all dependencies are also duplicated.